main.js

```diff
@@ -162,10 +162,134 @@ const inventory = [
+currentRevenue.style.color = "green";
+
+const typeOfTVOne = document.getElementById("typeOfTVOne");
+typeOfTVOne.textContent += inventory[1].type;
+
+const typeOfTVTwo = document.getElementById("typeOfTVTwo");
+typeOfTVTwo.textContent += inventory[2].type;
+
+const revenueContainer = document.getElementById("revenueContainer");
+
+function formatInventoryArray(inventory) {
+    for (let i = 0; i < inventory.length; i++) {
+        const tvFormatted = document.createElement("p");
+        tvFormatted.setAttribute("Id", "tvFormatted" + i);
+        tvFormatted.setAttribute('style', 'white-space: pre;');
+        tvFormatted.textContent = formatTVName(inventory[i]) + " \n" +
+            formatPrice(inventory[i].price) + " \n" +
+            formatTVSize(inventory[i].availableSizes);
+        revenueContainer.appendChild(tvFormatted);
+    }
+}
+
+formatInventoryArray(inventory);
+
+function formatTVName(television) {
+    return television.brand + " " + television.type + " - " + television.name + " \r";
+}
+
+function recalculateScreenSizeToCM(sizeInInches) {
+    return (sizeInInches / 0.39370).toFixed(0);
+}
+
+function formatTVSize(televisionSize) {
+    let screenSizeString = "";
+    for (const televisionSizeKey in televisionSize) {
+        if (!(televisionSize[televisionSizeKey] === televisionSize[televisionSize.length - 1])) {
+            screenSizeString += televisionSize[televisionSizeKey] + " inches " + "(" + recalculateScreenSizeToCM(televisionSize[televisionSizeKey]) + " cm) | ";
+        } else {
+            screenSizeString += televisionSize[televisionSizeKey] + " inches " + "(" + recalculateScreenSizeToCM(televisionSize[televisionSizeKey]) + " cm)";
+        }
+    }
+    return screenSizeString;
+}
+
+function clearElements(inventory) {
+    for (let i = 0; i < inventory.length; i++) {
+        let child = document.getElementById("tvFormatted" + i);
+        revenueContainer.removeChild(child);
+    }
+}
+
+let priceSort = document.getElementById("priceSort")
+priceSort.addEventListener("click", function sortByPrice() {
+    clearElements(inventory);
+    formatInventoryArray(leastExpensiveToMostExpensiveArray);
+});
+
+let ambilightSort = document.getElementById("ambilightSort")
+ambilightSort.addEventListener("click", function ambilightSort() {
+    clearElements(inventory);
+    formatInventoryArray(hasAmbilightArray);
+});
+
+let soldOutSort = document.getElementById("soldOutSort")
+soldOutSort.addEventListener("click", function soldOutSort() {
+    clearElements(inventory);
+    formatInventoryArray(soldOutArray);
+});
+
+
+
+function formatPrice(televisionPrice) { //formatteert de prijs volgens de ingebouwde format methode
+    return new Intl.NumberFormat('nl-NL', {
+        style: 'currency', currency: 'EUR'
+    }).format(televisionPrice);
+}
+
+console.log(formatPrice(inventory[1].price));
+console.log(formatTVName(inventory[1]));
+
+const typeInventoryArray = inventory.map((televisions) => {
```

Comment:
Let op dat het om één tv object gaat, en dat het daarom dus niet `televisionS` is, maar `television` 😉